### DISCUSSION TITLE
INIT B unavailable?

### DISCUSSION THREAD

**Discussion by Wnthr** [flybywiresim/aircraft#2356] [Help] [answered]
When clicking INIT and then one of the side arrow keys, instead of the INIT B page I expected, I am greeted by a 'not available' message. Is the INIT B page available by some other means? Going to FUEL PRED shows the orange boxes for ZFW/ZFWCS, but when entering a value in there, nothing happens.

**Comment by Benjozork** [comment, marked as answer]
Did you start your engines ? INIT B is not available when that's the case, which is how it is IRL.

**Comment by Wnthr** [comment]
After engine start, this info should go into FUEL PRED, but as of this writing, it seems there is a fix that hasn't been merged yet, so starting cold and dark seems to be the only option right now.
